src/github-release.ts

```diff
@@ -47,21 +46,58 @@ export class GitHubRelease {
     this.releasePR = options.releasePR;
   }
 
-  async run(): Promise<GitHubReleaseResponse | undefined> {
-    const candidate = await this.releasePR.buildRelease();
-    if (!candidate) {
+  async createRelease(): Promise<
```

Comment:
[In a future PR](https://github.com/joeldodge79/release-please/compare/manifest3-prep-GitHubRelease-for-manifest...joeldodge79:manifest4-githubRelease?expand=1#diff-c7a7a2623965c79d7ae68302c86163cb12a067a43756d8db6a213a316d39473aR528) it's called by the manifest releaser which supplies the `version` and `mergedPR` arguments.

Otherwise it's called by [`run()`](https://github.com/googleapis/release-please/pull/807/files/54690470c956703ca32e20ec5deb1eac6f712d1e#diff-cf4de6caacd9df882507f258826e7c79bf0f35a41670f5140ad91dfad917685dR96) w/out any arguments meaning they have to be looked up.